scripts/mkcloud

```diff
@@ -839,6 +843,7 @@ function install_vbmc
     zypper addrepo --priority 500 --refresh \
         http://download.suse.de/ibs/SUSE:/SLE-12-SP4:/Update:/Products:/Cloud9/standard/ \
         SLE-12-SP4-Cloud9
+
```

Comment:
I guess this was by mistake?

Reply:
yes and no. the lines below were in the beginning of this function as I didn't want to miss the case where vbmc is installed but those deps are not. I've just realized that `vbmc --version` would fail in such cases and the additional deps can be safely installed after the checks.